_posts/2018-06-18-On-Obsessive-Statelessness.markdown

```diff
@@ -183,22 +183,22 @@ expire the token at runtime.
 
 The user/authentication JWT is not the only JWT in play in Peril.
 
-I needed the ability for the JavaScript Runtime to send messages back to the Peril server. Peril re-uses [JWTs for
-creating a short-lived (2 min) token][jwt_two]. This token only has access to a single installation and is given a
+I needed the ability for the JavaScript Runtime to send messages back to the Peril server. Peril re-uses JWTs for
+creating a short-lived (2 min) token. This token only has access to a single installation and is given a
```

Comment:
I yanked the link since it wasn't found, but if you can remember what you were trying to link to, I can easily add it back in. ❤️

Reply:
makes sense, yeah